themes/cardinal_service/lib/scss/spotlight/spotlight.scss

```diff
@@ -0,0 +1,168 @@
+@charset "UTF-8";
+
+.spotlight-all-content-container {
+	display: flex;
+	align-items: center;
+	margin: 0 auto;
+	padding: 70px 0 0;
+
+	.image {
+		display: flex;
+		justify-content: flex-end;
+	}
+
+	img {
+		border-radius: 50%;
```

Comment:
just as an FYI, you can accomplish circle images via the image styles. but this is an alternative and works.